app/src/command.c

```diff
@@ -18,9 +22,30 @@ static inline const char *get_adb_command() {
     return adb_command;
 }
 
+static void show_err_msg(int err) {
+#ifdef __WINDOWS__
+    LOGE("Failed to execute adb.\n");
+#else
+    switch (err) {
```

Comment:
(indent cases for consistency, it makes it more readable when cases have blocks)